.babelrc

```diff
@@ -1,5 +1,19 @@
 {
-  "presets": ["es2015"],
+  "presets": [
+    ["env", {
+      "debug": true,
+      "targets": {
+        "node": 4.7,
+        "browsers": [
```

Comment:
same [comment](https://github.com/contentful/contentful-import/pull/18#discussion_r105629475) as in the contentful-import PR